packages/ui/v2/core/Shell.tsx

```diff
@@ -417,16 +417,32 @@ const navigation: NavigationItemType[] = [
     icon: Icon.FiGrid,
     isCurrent: ({ router, item }) => {
       const path = router.asPath.split("?")[0];
-      return !!item.child?.some((child) => path === child.href);
+      // During Server rendering path is /v2/apps but on client it becomes /apps(weird..)
+      return (
+        (path.startsWith(item.href) || path.startsWith("/v2" + item.href)) && !path.includes("routing_forms/")
+      );
     },
     child: [
       {
         name: "app_store",
         href: "/apps",
+        isCurrent: ({ router, item }) => {
```

Comment:
We won't need this after v2 launch

Reply:
Also from @hariombalhara on his contribution to my PR merged to this one a few days back